ngraph/test/models/onnx/onnx_prototxt_converter.py

```diff
@@ -53,6 +54,18 @@ def _is_txt_file(path):
 def _get_output_file_path(path, extension):
     return path + _ext_map[extension]
 
+
+def save_model(proto, f, format=None, save_as_external_data=False, all_tensors_to_one_file=True, location=None, size_threshold=1024, convert_attribute=False):
+    if isinstance(proto, bytes):
+        proto = onnx._deserialize(proto, onnx.ModelProto())
+
+    if save_as_external_data:
+        convert_model_to_external_data(proto, all_tensors_to_one_file, location, size_threshold, convert_attribute)
+
+    s = onnx._serialize(proto)
+    onnx._save_bytes(s, f)
```

Comment:
Why do we need this change?
Why we can't use a onnx library save mechanism?

Reply:
It doesn't work correctly for models which use external data feature.